api/app/Exceptions/ValidationException.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace App\Exceptions;
+
+use Illuminate\Support\MessageBag;
+use Illuminate\Http\Exception\HttpResponseException;
+
+class ValidationException extends HttpResponseException
```

Comment:
@xiphiaz 95bfdc0 adds this change. I added a call to ```ValidationException``` from ```BaseController.php:renderException``` that modifies the response so we get the invalid key.